test/asynchronous/helpers.py

```diff
@@ -404,5 +404,8 @@ def is_alive(self):
 
     async def run(self):
         if self.target:
-            await self.target()
+            if self.args:
+                await self.target(*self.args)
+            else:
+                await self.target()
```

Comment:
`await self.target(*self.args)` will work even if self.args is an empty list so this code can be simplified.